apps/oc_chef_wm/src/oc_chef_wm_authn_ldap.erl

```diff
@@ -115,7 +135,7 @@ bind(Session, BindDN, BindPassword) ->
     case eldap:simple_bind(Session, BindDN, BindPassword) of
         ok -> ok;
         {error, Error} ->
-            lager:error("Could not bind as ~p, please check private-chef.rb for correct bind_dn, bind_password, host, port and encrpytion values. Error: ", [BindDN, Error]),
+            lager:error("Could not bind as ~p, please check private-chef.rb for correct bind_dn, bind_password, host, port and encrpytion values. Error: ~p", [BindDN, Error]),
```

Comment:
BindDN won't be blank here.  It will be anon, so I think this will read:

"Could not bind as anon..."